### PR TITLE
Refactor reviews fetching (move to App.jsx)

### PR DESCRIPTION
**Pull request by tchitrakorn**
- Moved reviews fetching from RatingsReviews to App.jsx
- Set reviews property in state
- Passed reviews to RatingsReviews and reviewsNumber to ProductDetails

### CODE REVIEW

**Comment by tchitrakorn** [comment]
Hi @alizehrehman @matthewwrobel @jabrake could someone please review my PR? Tagging @alizehrehman as well just in case you would like to take a look yourself to confirm that this is what you need. Please let me know if it's not :)

File: client/ratings_reviews/RatingsReviews.jsx [hunk 1]
```diff
@@ -38,19 +38,11 @@ class RatingsReviews extends React.Component {
...
-    }).then((reviews) => {
-      this.setState({
-        meta: this.props.meta,
-        reviews: reviews.results,
-        filteredReviews: helpers.sortReviews(reviews.results, this.state.sortingOption)
-      });
-    }).catch((error) => {
-      console.log(error);
+    this.setState({
+      meta: this.props.meta,
```
Comment (alizehrehman) [inline client/ratings_reviews/RatingsReviews.jsx:42]: Something to consider: do you need this data that is now available through props in the state?

**Review by alizehrehman — APPROVED** [review]
Looks great! Left one comment to consider :)

File: client/ratings_reviews/RatingsReviews.jsx (hunk 1 above)
Comment (tchitrakorn) [reply client/ratings_reviews/RatingsReviews.jsx:42]: Will keep this in mind when refactoring!

**Pull request merged**